dotnet/src/Connectors/Connectors.AzureAIInference.UnitTests/Core/ChatClientCoreTests.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.AI.Inference;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Connectors.AzureAIInference.Core;
+using Microsoft.SemanticKernel.Http;
+using Microsoft.SemanticKernel.Services;
+using Moq;
+using Xunit;
+
+namespace SemanticKernel.Connectors.AzureAIInference.UnitTests.Core;
+
+public sealed class ChatClientCoreTests
+{
+    private readonly Uri _endpoint = new("http://localhost");
+
+    [Fact]
+    public void ItCanBeInstantiatedAndPropertiesSetAsExpected()
+    {
+        // Act
+        var logger = new Mock<ILogger<ChatClientCoreTests>>().Object;
+        var breakingGlassClient = new ChatCompletionsClient(this._endpoint, new AzureKeyCredential("key"));
+
+        var clientCoreModelConstructor = new ChatClientCore("model1", "apiKey", this._endpoint);
+        var clientCoreBreakingGlassConstructor = new ChatClientCore("model1", breakingGlassClient, logger: logger);
+
+        // Assert
+        Assert.NotNull(clientCoreModelConstructor);
```

Comment:
This assert, and the one below, will never fail unless there's an issue with the `new` operator.